src/lib/TwaManifest.ts

```diff
@@ -120,6 +121,7 @@ export class TwaManifest {
     this.packageId = data.packageId;
     this.host = data.host;
     this.name = data.name;
+    this.launcherName = data.launcherName || data.name;
```

Comment:
Is `|| data.name` necessary? We already have the logic for that down on line 208.

Reply:
Oh, I see now that is for the update case.

Sorry for the noise (but do you mind adding a comment saying why we're doing it?).

Reply:
Sure! 😄